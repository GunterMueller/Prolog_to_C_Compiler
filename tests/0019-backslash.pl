main:-
	display('this\tis \"a\"\ntest\n'),
	display("\'list\\\n"), nl.
