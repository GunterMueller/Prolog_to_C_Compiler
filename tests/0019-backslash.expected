this	is "a"
test
[39,108,105,115,116,92,10]
